pkg/scheduler/framework/interface_test.go

```diff
@@ -27,13 +27,14 @@ import (
 
 var errorStatus = NewStatus(Error, "internal error")
 
-func TestIsSuccessStatus(t *testing.T) {
+func TestStatus(t *testing.T) {
 	tests := []struct {
 		name              string
 		status            *Status
 		expectedCode      Code
 		expectedMessage   string
 		expectedIsSuccess bool
+		expectedIsWait    bool
```

Comment:
we made these changes because we thought it would be better to simply add Test Status status rather than writing generic test code.